x-pack/plugins/siem/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -267,7 +301,41 @@ const getAuthenticationColumns = (startDate: number): Array<Columns<Authenticati
     name: i18n.SUCCESSES,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }) => defaultToEmptyTag(node.successes),
+    render: ({ node }) => {
+      const id = escapeDataProviderId(
+        `authentications-table-${node._id}-node-successes-${node.successes}`
+      );
+      return (
+        <DraggableWrapper
+          key={id}
+          dataProvider={{
+            and: [],
+            enabled: true,
+            id,
+            name: 'authentication_success',
+            excluded: false,
+            kqlQuery: '',
+            queryMatch: {
+              field: 'event.type',
+              value: 'authentication_success',
```

Comment:
You'll want the actual value here, no? So `node.successes`? Also, is `event.type` the correct field?

Reply:
So you can filter on all success or failure but you can't do complex queries yet such as , `user.name: "frank" and event.type: "authentication_success"`